### PR TITLE
Relic

### PR DESCRIPTION
**Pull request by shogi880**
## What does this PR do?

Fixes # (issue)

## Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/lightning-bolts/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? \[not needed for typos/docs\]
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/lightning-bolts/blob/master/CHANGELOG.md)?

## PR review

- [x] Is this pull request ready for review? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: pl_bolts/models/self_supervised/simclr/simclr_module.py [hunk 1]
```diff
@@ -80,6 +80,7 @@ def __init__(
         learning_rate: float = 1e-3,
         final_lr: float = 0.0,
         weight_decay: float = 1e-6,
+        relic: bool = False,
```
Comment (Borda) [inline pl_bolts/models/self_supervised/simclr/simclr_module.py:83]:
```suggestion
        use_relic_loss: bool = False,
```

**Comment by Borda** [comment]
@shogi880 @ananyahjha93 can you pls clarify what is this contribution about?
@shogi880 I see some issue, you are changing almost all repo ~250 files

**Comment by Borda** [comment]
@shogi880 any update here? :rabbit: 
cc: @SeanNaren otherwise I am for closing it... :confused:

**Comment by shogi880** [comment]
Hi @Borda 
Sorry for the delay.
I'm new to GitHub and programming, so I think I've included a lot of log files.
I'll close it and request again after I clean it up.

**Comment by shogi880** [comment]
btw, really thanks for this perfect library that is organized by you.

**Pull request closed without merging**